tests/phpunit/tests/blocks/renderCommentTemplate.php

```diff
@@ -187,9 +186,6 @@ function test_build_comment_query_vars_from_block_pagination_with_no_comments()
 			),
 			build_comment_query_vars_from_block( $block )
 		);
-
-		update_option( 'comments_per_page', $comments_per_page );
-		update_option( 'default_comments_page', $default_comments_page );
```

Comment:
Is the option table included in the transaction that gets rolled back? For some reason I thought it wasn't.

If it is, then the definition of these variables at the start of this test can be removed too. They are only used for these lines.

Reply:
🤦‍♀️  Duh, you're right. 🤦‍♀️ What was I thinking 🤣 

Will move these to the `tear_down()` to ensure the starting state is set between each test. Revision coming.

Reply:
Done here https://github.com/WordPress/wordpress-develop/pull/2701/commits/2e83e9e87be9fa8bcd52c9e1ce1517183f4094c3 👍